app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -216,17 +216,20 @@ function AddressSearch({
           hint={t('in_person_proofing.body.location.po_search.address_search_hint')}
         />
       </ValidatedField>
-      <SpinnerButton
-        isWide
-        isBig
-        ref={spinnerButtonRef}
-        type="submit"
-        className="margin-y-5"
-        onClick={handleSearch}
-        spinOnClick={false}
-      >
-        {t('in_person_proofing.body.location.po_search.search_button')}
-      </SpinnerButton>
+      <div className="margin-y-5">
```

Comment:
@allthesignals Could you add 1rem (8px) between the bottom of the button and the button message? It looks pretty tight in the PR video. Here's a screenshot from Figma:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/12240896/209876851-ff139812-b251-43a4-ae26-9df15bc07702.png">

Reply:
nice! looks good to me!